app/views/provisioning_templates/_alerts.html.erb

```diff
@@ -1,5 +1,6 @@
 <% if @template.locked? -%>
   <%= locked_warning(::Template.find(params[:id])) %>
 <% else -%>
+  <%= infobox_safemode if Setting[:safemode_render] %>
   <%= infobox_functions_and_macros("4.4.3ProvisioningTemplates") %>
```

Comment:
This can move onto the Help tab, either as a block of text or alert at the top of the tab, it'd be a much better place for it.